### DISCUSSION TITLE
Python與微積分—繪製函數圖形

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#368] [General]
在使用微積分繪製函數(y=f(x))圖形時，首先要計算臨界點(critical point)。臨界點就是讓函數的一階導數(微分)=0或不存在(**d**oes **n**ot **e**xist, DNE)的點。臨界點**不是**讓函數=0的自變數點！計算臨界點的原因在於一階導數代表函數圖形的斜率(slope)，斜率=0可能是函數圖形轉彎的地方(可能是代表可能不是)，斜率不存在的地方，也可能是函數圖形轉彎的地方(可能是代表可能不是)。而圖形要怎麼轉彎，就是要畫出圖形的一部份。

**Comment by enoch-lightning** [comment]
函數的一階導數代表斜率，也就是代表圖形在自變數增加的情況下，會函數會變大還是變小。若斜率(一階導數)大於0，代表在自變數增加的情況下，函數會變大；若斜率(一階導數)小於0，代表在自變數增加的情況下，函數會變小；若斜率(一階導數)=(等於)0，代表在自變數增加的情況下，函數不會變。總而言之，一階導數代表圖形會遞增(函數變大)還是遞減(函數變小)。

**Comment by enoch-lightning** [comment]
知道臨界點與一階導數大於0(+)、等於0，還是小於0(-)後，就知道函數80%的長相。因為藉由臨界點與一階導數，我們可以得到函數隨自變數增加而變大或變小的圖。

**Comment by enoch-lightning** [comment]
若圖形還想畫得更精美，這時候就要可慮二階導數與漸近線。先計算二階導數來決定函數圖形在哪裡要向上凹，在哪裡要向下凹。最後，再由是否有斜漸近線、是否有垂直漸近線與是否有水平漸近線的順序求出並畫出所有的漸近線。

**Comment by enoch-lightning** [comment]
# 漸近線
名詞翻譯：
單數(一條)：asymptote
複數(兩條=二條、三條、四條、五條或更多條)：asymptotes

問題什麼是漸近線？
答案函數圖形 _**漸漸靠近的線**_。

漸近線分類
- 水平漸近線

1. 名詞翻譯

- 單數horizontal asymptote
- 複數horizontal asymptotes
- 垂直漸近線

2.名詞解釋
- 水平的漸近線 

1.名詞翻譯

- 單數vertical asymptote

- 複數vertical asymptotes

2.名詞解釋
-垂直的漸近線 

- 斜漸近線

1. 名詞翻譯

- 單數oblique asymptote
- 複數oblique asymptotes
2. 名詞解釋
- 斜的漸近線

求出漸近線順序
第一步判斷是否有斜漸近線(看函數就看得出來)
第二步判斷是否有垂直漸近線(看函數就看得出來)
第三步判斷是否有水平漸近線(一定要計算正負無限大(無窮大)的極限才看得出來。)

我的標點符號鍵盤顯示不出來。所以上面的冒號我都打不出來。
我不知道要如何縮排，還在研究。

**Comment by pekoranoinu** [comment]
這部分我剛好要用到，謝謝你的解釋。
